converter_test.go

```diff
@@ -1218,9 +1511,16 @@ func TestArrowToRecord(t *testing.T) {
 			meta := tc.rowType
 			meta.Type = tc.logical
 
-			transformedRec, err := arrowToRecord(rawRec, pool, []execResponseRowType{meta}, localTime.Location())
+			transformedRec, err := arrowToRecord(rawRec, pool, []execResponseRowType{meta}, localTime.Location(), tc.orgTS)
 			if err != nil {
+				if strings.Contains(err.Error(), tc.error) {
+					return
```

Comment:
Maybe we can negate if and omit this strange return? It would be best to write:
```
if !strings.contains(...) {
  t.Errorf("expected %v to contain %v")
}
```

Reply:
I reorganized it a little.